test/runtime-browser/browser.test.js

```diff
@@ -163,11 +159,111 @@ describe(
 
 		await Promise.all(
 			fs.readdirSync(`${__dirname}/samples`).map(async (dir) => {
-				await runTest(dir, false);
-				await runTest(dir, true);
+				await run_test(dir, false);
+				await run_test(dir, true);
 			})
 		);
 	},
 	// Browser tests are brittle and slow on CI
 	{ timeout: 20000, retry: process.env.CI ? 1 : 0 }
 );
+
+describe(
+	'custom-elements',
+	async () => {
+		async function run_test(dir) {
```

Comment:
could you move this function outside the `describe`? it's so indented in the diff that almost every line wraps on a small screen, which makes it hard for me to read